source/vision/visionHandler.py

```diff
@@ -348,3 +356,9 @@ def initialFocus(self) -> None:
 			# focus/review hasn't yet been initialized.
 			return
 		self.handleGainFocus(api.getFocusObject())
+	
+	def initialNavigatorObject(self) -> None:

```

Comment:
I think this should be named "initializeNavigatorObject". Could you add a brief docstring?

```suggestion
	def initializeNavigatorObject(self) -> None:
```

Reply:
I have named `initialNavigatorObject` as done per `initialFocus`. If you want me to rename one, it would make sense to rename both. But in this case, it would break the API, wouldn't it? I am a bit concerned to have to break the API just for a better name. In this case, I would prefer adding an explicit docstring to both functions. What do you think?
If you maintain that these functions should be renamed however, I feel that `handleInitialNavigatorObject` would be more suitable than `initializeNavigatorObject`, since the provider does not initialize the nav object; it rather handles the specific case of the initial nav object when it starts.

I have added the docstrings in d7506dcaa7814319f2af6873aec499fe7b2ac262; feel free to suggest rephrasing if you find better.

Reply:
Nevermind - these names seem appropriate.